_content/articles/dharma-songs-in-contemporary-cambodia_walker-trent.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Saṃvega and Pasāda: Dharma Songs in Contemporary Cambodia"
 authors:
-  - "Trent Walker"
-external_url: "https://poj.peeters-leuven.be/content.php?url=article.php&id=3285745&journal_code=JIABS"
+  - "walker"
+external_url: "https://doi.org/10.2143/JIABS.41.0.3285745"
```

Comment:
Please change all the other files authored by Trent Walker to the slug in this same branch/PR